packages/components/src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,194 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconCross } from '@frontify/fondue-icons';
+import * as RadixDialog from '@radix-ui/react-dialog';
+import { forwardRef, type CSSProperties, type ForwardedRef, type ReactNode } from 'react';
+
+import {
+    dialogBodyStyles,
+    dialogContentStyles,
+    dialogFooterStyles,
+    dialogHeaderStyles,
+    dialogSideContentStyles,
+    dialogUnderlayStyles,
+} from './styles/dialogStyles';
+
+export type DialogRootProps = {
+    /**
+     * Disable interaction with the rest of the page
+     * @default false
+     */
+    modal?: boolean;
+    /**
+     * The controlled `open` state of the Dialog
+     */
+    open?: boolean;
+    /**
+     * Event handler called when the `open` state changes
+     */
+    onOpenChange?: (open: boolean) => void;
+    children?: ReactNode;
+};
+
+export type DialogContentProps = {
+    /**
+     * Add rounded corners to the Dialog
+     * @default true
+     */
+    rounded?: boolean;
+    /**
+     * Define the padding of the Dialog
+     * @default "compact"
+     */
+    padding?: 'compact' | 'comfortable' | 'spacious';
+    /**
+     * Define a maximum width for the Dialog
+     * @default 800px
+     */
+    maxWidth?: string;
+    /**
+     * Define a minimum width for the Dialog
+     * @default 400px
+     */
+    minWidth?: string;
+    /**
+     * Define a maximum height for the Dialog
+     * @default 200px
+     */
+    minHeight?: string;
+    /**
+     * Show a dark underlay behind the dialog
```

Comment:
either lowercase or uppercase, not both.
(preferrence for the lowercased)

Reply:
changed